service/api_logic/models/api_models.py

```diff
@@ -26,3 +26,11 @@ def __init__(self):
         self.related_name = "name"
         self.related_logo = "logo"
         self.related_id = "team_index_id"
+
+
+class InteractionTypes(Enum):
```

Comment:
We should have the exact same thing on our frontend (to avoid hardcoding strings like I mentioned above). Maybe make a cachable API call to retrieve this to frontend